src/libs/toggleReportActionComposeView/index.js

```diff
@@ -0,0 +1 @@
+export default () => {};
```

Comment:
So I'm noticing that you're differentiating between web and desktop to determine whether or not we should toggle the compose input. Instead, I think we should make that distinction based on whether the device has a small screen width. That way this will work the same on mobile web as on iOS/Android

Reply:
Makes sense!